packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/KycVerification/ExtraFields/template.success.tsx

```diff
@@ -157,6 +157,28 @@ const LabelItem = styled.label`
 const Success: React.FC<InjectedFormProps<{}, Props> & Props> = (props) => {
   const disabled = props.invalid || props.submitting
 
+  // due to default invalid state of redux-form we need to check if all multi-select nodes are checked
+  // and if they are checked we need to click on them to update redux-form state and remove invalid state
+  useEffect(() => {
```

Comment:
Does this need to be an effect (that runs after first render) or can it be achieved all at once before rendering?

Also - can't the `redux-form` state be modified before this? When setting it up or something?